apps/global/i18n/translations/es.json

```diff
@@ -106,6 +106,11 @@
     "meetTheCollectiveTitle": "Conoce a la colectiva",
     "meetTheCollectiveDescription": "Dream On: Speaker Collective es un equipo de autores, oradores y predicadores. Nos apasiona empoderar y enseñar a otros a perseguir sus sueños y propósitos que Dios ha puesto en la vida de todos y cada uno."
   },
+  "board": {
```

Comment:
You will need to update these translations to reflect the english changes I recommended above